addons/mail/static/src/components/chatter_topbar/chatter_topbar.js

```diff
@@ -98,9 +98,10 @@ class ChatterTopbar extends Component {
         return this.env.bus.trigger('do-action', {
             action,
             options: {
-                // A bit "extreme", could be improved:
-                // normally only an activity is created (no update nor delete)
-                on_close: () => this.chatter.refreshActivities(),
+                on_close: () => {
+                    this.chatter.refreshActivities();
+                    this.chatter.refresh();
+                },
```

Comment:
`mail.chatter` code is kinda crap, but that's that and planned clean/re-design (task-2243518) so let's not bother with this now.
Why remove the comment? I still think it's relevant.

Reply:
That's because there is the same in `refreshActivities`, so I think the one there is relevant and sufficient, no ?

Reply:
ok